share/spack/gitlab/cloud_pipelines/stacks/e4s-cray-sles-rocmcc/spack.yaml

```diff
@@ -0,0 +1,68 @@
+spack:
+  include:
+  - $spack/share/spack/gitlab/cloud_pipelines/configs/linux/ci.yaml
+
+  view: false
+
+  concretizer:
```

Comment:
The concretizer configs should be coming from the common CI configs.

Reply:
@kwryankrattiger Do you mean `configs/concretizer.yaml`?  If so, I'm not seeing where it is included.

Reply:
Or do you mean it should be pasted into `configs/ci.yaml`?

Reply:
Nvm.  I see use of `--config-scope` in `.generate-common`'s `script`.